src/draws/drawAxisComponents.js

```diff
@@ -38,10 +38,7 @@ export default (function drawAxisComponents(
       });
     });
     // only show the dendrogram if the current axis is in clust ordering
-    if (
-      store.select("order").inst[inst_axis] === "clust" &&
-      store.select("order").new[inst_axis] === "clust"
-    ) {
+    if (store.select("order").new[inst_axis] === "clust") {
```

Comment:
inst is the previous state of the button clicked and new is the current state of the button clicked, And dendrogram are added only when we select "clust" button. so this was removed so that it will only check if the new state or current state is clust or not.

Reply:
If this line was functional in the original codebase, the issue would more likely be in our implementation and not here.